benchopt/parallel_backends/tests/test_parallel_backends.py

```diff
@@ -77,7 +77,8 @@ def test_backend_collect(backend):
             rm_modules=['parallel_backends', 'submitit', 'distributed', 'dask']
     ):
         with temp_benchmark(config={"parallel_config.yml": config}) as bench:
-            cmd = f"{bench.benchmark_dir} -d test-dataset -n 0 -r 1 --no-plot"
+            cmd = f"{bench.benchmark_dir} -d test-dataset -n 0"
+            cmd += " -r 1 --no-plot --seed 0"
```

Comment:
Don't you need to use the same seed in the `--collect`? Otherwise, smthg is broken here ^^
I guess as the benchmark does not use `seed`, this extra args is actually not needed?